cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -159,9 +159,10 @@ def _hosts_remove_creds():
 
 def set_agent_creds(appliance, request, provider):
     version = appliance.version.vstring
-    agent_data = {"ems": {"ems_amazon": {
-        "agent_coordinator": {"docker_image": "simaishi/amazon-ssa:{}".format(version),
-                              "docker_registry": "docker.io"}}}}
+    docker_image_name = "simaishi/amazon-ssa: {}".format(version)
```

Comment:
**REQUIRED** why is a space introduced in the docker image name?
**QUESTION** why not let prt run the related test?

Reply:
As the URL is long and affecting lint, so generated link under `Docker_image_name`.
PRT will failed as SmartState docker creds update require some amount of time, PR #7523 is raised to fix it.

Reply:
i meant
```diff
- "simaishi/amazon-ssa:{}"
+ "simaishi/amazon-ssa: {}"
``` i believe that is a incorrect change

Reply:
oops!! I will update it.